core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -1266,6 +1311,233 @@ class UserServiceSpec
       .get shouldEqual CloudPlatform.UNKNOWN.toString
   }
 
+  /* it should "not attempt to update landing zones for non-azure projects" in {
```

Comment:
@blakery why is this one commented out?

Reply:
Because I didn't clean up after myself and remove it
And because it was happening as a side effect, before I updated `updateLandingZoneStatus` to use a future, and it was more documentation then test condition